test/rulesets/Base/fastmath_able.jl

```diff
@@ -125,8 +125,8 @@ const FASTABLE_AST = quote
 
     @testset "binary functions" begin
         @testset "$f(x, y)" for f in (atan, rem, max, min)
-            test_frule(f, 100rand(), 10rand())
-            test_rrule(f, 100rand(), 10rand())
+            test_frule(mod, rand_mod_pair()...)
+            test_rrule(mod, rand_mod_pair()...)
         end
```

Comment:
Typo: `f` not `mod` 

(Not sure why GitHub won't let me add suggestion)

Reply:
Oops, bad copy-paste!